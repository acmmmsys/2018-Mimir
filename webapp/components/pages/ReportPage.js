import React, { Component } from 'react';

import Panel from 'layout/Panel';
import PanelHeader from 'layout/PanelHeader';
import ContentGrid from 'layout/ContentGrid';
import ContentColumn from 'layout/ContentColumn';
import HeaderGrid from 'layout/HeaderGrid';
import HeaderColumn from 'layout/HeaderColumn';
import ContentRow from 'layout/ContentRow';

import PanelReport from 'components/panels/PanelReport';


const ReportPage = () =>
    <div>
        <HeaderGrid>
            <HeaderColumn>
                <PanelHeader>
                </PanelHeader>
            </HeaderColumn>
        </HeaderGrid>
        <ContentGrid>
            <ContentColumn>
                <PanelReport />
            </ContentColumn>
        </ContentGrid>
    </div>

export default ReportPage;