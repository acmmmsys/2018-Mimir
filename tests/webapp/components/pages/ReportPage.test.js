import React from 'react';
import ReportPage from '../../../../webapp/components/pages/ReportPage'
import renderer from 'react-test-renderer';

let wrapper;

test('ReportPage does render', () => {
    wrapper = shallow(
        <ReportPage>

        </ReportPage>
    );
    expect(wrapper).toMatchSnapshot();
});
