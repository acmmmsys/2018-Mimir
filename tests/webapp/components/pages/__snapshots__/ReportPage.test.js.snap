// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportPage does render 1`] = `
<div>
  <HeaderGrid>
    <HeaderColumn>
      <defaultProps(branch(branch(PanelHeader)))
        emptyMessage=""
        loading={false}
      />
    </HeaderColumn>
  </HeaderGrid>
  <ContentGrid>
    <ContentColumn>
      <Connect(nest(defaultProps(branch(branch(Panel))), ColonoscopyReport)) />
    </ContentColumn>
  </ContentGrid>
</div>
`;
